src/test/java/me/sat7/dynamicshop/transactions/TransactionTests.java

```diff
@@ -33,4 +34,14 @@ public void calcCostShouldBeTheSameWhenInStock() {
         assertEquals(amountSell, amountBuy, 0.01);
     }
 
+    @Test
+    public void staticMedianOrStockShouldCalcPrice() {
+        ShopUtil.ccShop.get().set("default.COBBLESTONE.stock", -1);
+        ShopUtil.ccShop.get().set("default.COBBLESTONE.median", -1);
+        double amountSell = Calc.calcTotalCost("default", "COBBLESTONE", -3);
+        assertEquals(30000, amountSell, 0.01);
+        double amountBuy = Calc.calcTotalCost("default", "COBBLESTONE", 3);
+        assertEquals(30000, amountSell, 0.01);
+    }
```

Comment:
Interesting. Is there a test-only shop?
In general, this test is likely to fail due to tax.